packages/vue/src/components/primitives/base-select.vue

```diff
@@ -14,7 +14,7 @@
 <script setup lang="ts">
 import { toRefs } from 'vue';
 const props = withDefaults(
-  defineProps<{ selectValue: string; options: string[] }>(),
+  defineProps<{ selectValue: string; options: string[] | undefined }>(),
```

Comment:
Upgrading eslint surfaced this issue because `dialCodeList` could be undefined. @wlee221 pls confirm this is fine :)

Reply:
This is an internal primitive that is not exported, so LGTM 👍 

https://github.com/aws-amplify/amplify-ui/blob/main/packages/vue/__tests__/exports.spec.ts

Reply:
Yeah, should be fine since its an addition